content/resources/Library/BabylonJS_and_Poki.md

```diff
@@ -0,0 +1,73 @@
+---
+title: BabylonJS and Poki
+image: 
+description: Using babylon.js to create a game for Poki.
+keywords: babylon.js, extension, external libraries, external, Poki
+further-reading:
+video-overview:
+video-content:
+---
+
+Poki.com is a curated discovery platform for free, high-quality online games. You can reach millions of players around the globe and get a revenue share from advertising when your game is played.
+
+This documentation explains how to transform an existing Babylon.js game to make it available on Poki.com 
+
+## Useful links
+
+* [Poki.com](https://poki.com/)
+* [Poki developers page](https://developers.poki.com/)
+* [Poki SDK documentation](https://sdk.poki.com/)
+
+
+## Some Babylon.js games on Poki
+
+* [Minecraft Classic - By Mojan](https://poki.com/en/g/minecraft-classic)
+* [Tunnel Rush - By Deer Cat](https://poki.com/en/g/tunnel-rush)
+* [Shell Shockers - By Blue Wizard Digital](https://poki.com/en/g/shell-shockers)
+
+
+## The process
+
+#### 1) Include the Poki Javascript SDK
+
+Add the following HTML within the ```<head>``` tags of your game HTML:
+```html
+    <script src="//game-cdn.poki.com/scripts/v2/poki-sdk.js"></script>
+```
+
+#### 2) Initialize the SDK 

```

Comment:
Want to change that to a 3rd level heading? Then it will appear on the ToC.

Same goes to the rest of the 4-level headings. If done intentionally, it's all fine.

Reply:
3rd level were too big, so I used 4th level. Let's keep like this if possible. Thanks!